tests/conftest.py

```diff
@@ -93,6 +93,6 @@ def _teardown_cached_dir(directory: Path | str) -> Path:
 @pytest.fixture(scope="session")
 def make_commit_obj() -> MakeCommitObjFn:
     def _make_commit(message: str) -> Commit:
-        return Commit(repo=Repo(), binsha=Commit.NULL_BIN_SHA, message=message)
+        return Commit(repo=Repo(), binsha=Commit.NULL_BIN_SHA, message=message)  # noqa: F405
```

Comment:
wasn't sure about this one, not sure if there's a better fix vs. ignoring?

Reply:
I don't even know what is wrong with this line.

F405: https://docs.astral.sh/ruff/rules/undefined-local-with-import-star-usage/

Reply:
The full error is:
```
F405 `Repo` may be undefined, or defined from star imports
```

I think because `Repo()` isn't imported / defined directly in `conftest.py`? Is it getting it implicitly from somewhere else (another import)?

Reply:
Ah ok, yeah that is it. Just add to line 10, `from git import Commit, Repo`. It was not intended result

Reply:
Yeah, wasn't sure if it was just really a missing import. Thanks for the heads up - added that in and removed the ignore.